README.md

```diff
@@ -320,20 +320,20 @@ It is not a good practice to use one of the following versioning styles:
   - `brand-v<datetime>.js` 
   - `brand-v1-v2.js`,
 
-The reason behind this is that It becomes confusing to track the lastest version. 
-Therefore, previous styling do not help developers who use the SDK.
+The reason is that it becomes confusing to track the lastest version. 
+Therefore, previous styling does not help developers who use the SDK.
 
-It is however a good practice to use [Semantic Versioning](http://semver.org), a.k.a **SemVer**, when versioning SDKs.
-It worths to mention that SemVer has three main parts, each corresponding to importance of a release: "MAJOR.MINOR.PATCH".
-Version in `v1.0.0` `v1.5.0` `v2.0.0` is easier to trace and track in changelog documentations, for instance.
+It is however a good practice to use [Semantic Versioning](http://semver.org), a.k.a. **SemVer**, when versioning SDKs.
+It's worth mentioning that SemVer has three main parts, each corresponding to importance of a release: "MAJOR.MINOR.PATCH".
```

Comment:
"It's worth mentioning" ~ 

The right way is "It worth mentioning". Alternatively, "It worth to mention" can do.

Reply:
I will vote for removing all, just use 

> It has three main parts

the previous sentence already has `It is however a good practice`, so not really needed to add an extra underline to the later sentence which drawing the same picture.

Reply:
I don't think that's correct. but whatever you guys wants.